Entities/HDFirearm.cpp

```diff
@@ -787,7 +787,7 @@ void HDFirearm::Update()
                         (m_Supported ? 1.0F : m_NoSupportFactor) * RandomNormalNum();
                 tempNozzle = m_MuzzleOff.GetYFlipped(m_HFlipped);
                 tempNozzle.DegRotate(degAimAngle + shake);
-                roundVel.SetIntXY(pRound->GetFireVel(), 0);
+                roundVel.SetXY(static_cast<int>(pRound->GetFireVel()), 0);
```

Comment:
Pretty sure the cast here and on line 854 aren't necessary.

Reply:
Right, this stuff conformed to a previous revision. Removing it.